views/account/profile/style.css

```diff
@@ -90,3 +90,48 @@ input {
 select {
   max-width: fit-content;
 }
+
+/* Modal CSS */
```

Comment:
If the modal css is going to be reused or copy/pasted in different files, we should just move it to the common.css file

Reply:
The same could be said for the js code if there's an easy way to do it.

Reply:
i just moved the modal css into the common.css. im going to look into if i can do the same thing for the js, but i think its usually specific to the webpage. lmk if you have any ideas

Reply:
For the js side we can try making a partial file and importing it to the pages that have a modal.

Reply:
It'd be similar to the defaultsettings partial